src/containers/admin-dashBoard/tests/ManageRole.test.tsx

```diff
@@ -16,9 +16,9 @@ describe('Manage role page tests', () => {
         <ManageRole />
       </BrowserRouter>,
     );
-    let removeManageRole = getByTestId('remove');
-    fireEvent.click(removeManageRole);
-    expect(removeRoleModel).toHaveBeenCalledTimes(0);
+    // let removeManageRole = getByTestId('remove');
+    // fireEvent.click(removeManageRole);
+    // expect(removeRoleModel).toHaveBeenCalledTimes(0);
```

Comment:
Since this codeblock is not needed, you can remove it.

Instead of commenting out the codeblock if you have added changes that removes them.